go.mod

```
@@ -8,7 +8,7 @@ require (
 	github.com/flynn/go-shlex v0.0.0-20150515145356-3f9db97f8568 // indirect
 	github.com/gliderlabs/ssh v0.1.1 // indirect
 	github.com/google/go-cmp v0.5.4 // indirect
-	github.com/gookit/color v1.3.2
+	github.com/gookit/color v1.3.9-0.20210406063659-9f9107323cbe
```

It looks like 1.4.2 released yesterday:
https://github.com/gookit/color/releases/tag/v1.4.2
```suggestion
	github.com/gookit/color v1.4.2
```